content/document-paths.ts

```diff
@@ -85,6 +85,9 @@ function childrenForPath(
 }
 
 function initAllDocumentsPathsTree(): Tree {
+  if (!CONTENT_ROOT) {
+    throw new Error("Environmental variable CONTENT_ROOT has not been set");
```

Comment:
We could be consistent with `tool/cli.ts`, and also add a hint to the .env/.env-dist files:

```suggestion
    throw new Error("CONTENT_ROOT not set. Please add it to your .env file (see the .env-dist file).");
```